core/templates/services/literally-canvas-helper.service.ts

```diff
@@ -0,0 +1,308 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview LiterallyCanvas editor helper service.
+ */
+
+const CONSTANTS = require('constants.ts');
```

Comment:
```suggestion
const constants = require('constants.ts');
```

Reply:
Ah, I think this is done because of a name conflict. It should be fixed by #9186. @rohitkatlaa can do something this here for now. Thanks!
https://github.com/oppia/oppia/blob/4a59747d464056fa58fa3a6af30a5d6310bfcf2a/core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts#L37-L39

Reply:
Done.